src/wallet/wallettool.cpp

```diff
@@ -183,6 +183,79 @@ static bool DumpWallet(std::shared_ptr<CWallet> wallet)
     return ret;
 }
 
+static bool CreateFromDump(const std::string& name, const fs::path& wallet_path)
```

Comment:
In commit "wallettool: Add createfromdump command" (7ea3df8ccd13096088538fb4602208eae0831c13)

Would suggest moving this to src/wallet/dump.cpp similar to src/wallet/salvage.cpp so the dump /load functionality be accessed other places outside the wallet tool, like by RPCs, the GUI, and c++ unit tests. Way arguments passed and errors are returned might have to change a little, though

Reply:
I've moved it to a new file.